packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { Authorize } from '@deriv/api-types';
+import { Localize, localize } from '@deriv/translations';
+import { Icon, Money } from '@deriv/components';
+import { Text, Accordion, Button } from '@deriv/ui';
+import { ArrayElement } from 'Types';
+import WalletCard from 'Components/wallet';
+import WalletActionButton from 'Components/wallet-action-button';
+import AppsLauncher from 'Components/app-launcher';
+import { observer } from 'mobx-react-lite';
+
+type WalletAccountProps = {
+    account: ArrayElement<Required<Authorize>['account_list']>;
+};
+
+const WalletAccount = observer(({ account }: WalletAccountProps) => {
+    return (
+        <>
```

Comment:
@iman-fs Not sure why we need a fragment here, `Accordion` is the only child 🤔

Reply:
we needed it before because we had another element inside it. forgot to remove it